haskell/servant/servant-proj.cabal

```diff
@@ -0,0 +1,24 @@
+name:                servant-proj
+version:             0.1.0.0
+-- synopsis:
+-- description:
+category:            Web
+build-type:          Simple
+cabal-version:       >=1.10
+
+library
+  hs-source-dirs:      src
+  exposed-modules:     Lib
+  build-depends:       base >= 4.7 && < 5
+                     , servant-server
+                     , wai
+                     , warp
+  default-language:    Haskell2010
+
+executable servant-proj-exe
+  hs-source-dirs:      app
+  main-is:             Main.hs
+  ghc-options:         -threaded -rtsopts -with-rtsopts=-N -with-rtsopts=-A64M -O2 -funbox-strict-fields
```

Comment:
Could this options be directly passed to `stack build` @whiplash ?

Reply:
I think so. But why do you think it's better? `stack build --ghc-options="-O2"` should work

Reply:
I prefer to set those options on `Dockerfile`, in order to have the same compilation option, if other `haskell` **frameworks** comes :stuck_out_tongue_closed_eyes: